base_report_to_printer/i18n/fr.po

```diff
@@ -0,0 +1,984 @@
+msgid "Print attachments"
+msgstr "Imprimer les pièces jointes"
+
+#. module: base_report_to_printer
+#: model:ir.model,name:base_report_to_printer.model_printing_printer
+#: model:ir.model.fields,field_description:base_report_to_printer.field_printing_job__printer_id
+#: model:ir.model.fields,field_description:base_report_to_printer.field_printing_report_xml_action__printer_id
+#: model:ir.model.fields,field_description:base_report_to_printer.field_printing_tray__printer_id
+#: model:ir.model.fields,field_description:base_report_to_printer.field_wizard_print_attachment__printer_id
+msgid "Printer"
+msgstr "Imprimante"
+
+#. module: base_report_to_printer
+#: model:ir.model,name:base_report_to_printer.model_printing_tray
+msgid "Printer Tray"
+msgstr "Bac d'impression"
+
+#. module: base_report_to_printer
+#: model:ir.model.fields.selection,name:base_report_to_printer.selection__printing_job__job_state_reason__printer-stopped
+msgid "Printer is stopped"
+msgstr "L'imprimante est arrêtée"
+
+#. module: base_report_to_printer
+#: model:ir.model.fields.selection,name:base_report_to_printer.selection__printing_job__job_state_reason__job-queued-for-marker
+msgid "Printer needs ink/marker/toner"
+msgstr "L'imprimante a besoin d'encre/de marqueurs/de toner"
+
+#. module: base_report_to_printer
+#: model:ir.model.fields.selection,name:base_report_to_printer.selection__printing_job__job_state_reason__printer-stopped-partly
+msgid "Printer state reason set to 'stopped-partly'"
+msgstr "Le motif de l'état de l'imprimante est \"arrêté partiellement\""
+
+#. module: base_report_to_printer
+#: model:ir.model.fields,help:base_report_to_printer.field_printing_job__printer_id
+msgid "Printer used for this job."
+msgstr "Imprimante utilisée pour cette tâche."
+
+#. module: base_report_to_printer
+#: model:ir.model.fields,help:base_report_to_printer.field_wizard_print_attachment__printer_id
+msgid "Printer used to print the attachments."
+msgstr "Imprimante utilisée pour imprimer les pièces jointes."
+
+#. module: base_report_to_printer
+#: model:ir.ui.menu,name:base_report_to_printer.printing_printer_menu
+#: model_terms:ir.ui.view,arch_db:base_report_to_printer.printing_printer_view_form
+#: model_terms:ir.ui.view,arch_db:base_report_to_printer.printing_printer_view_search
+#: model_terms:ir.ui.view,arch_db:base_report_to_printer.printing_server_view_form
+msgid "Printers"
+msgstr "Imprimantes"
+
+#. module: base_report_to_printer
+#: model:ir.model.fields,field_description:base_report_to_printer.field_printing_server__printer_ids
+msgid "Printers List"
+msgstr "Liste des imprimantes"
+
+#. module: base_report_to_printer
+#: model:ir.model.fields.selection,name:base_report_to_printer.selection__printing_printer__status__printing
+#: model:ir.ui.menu,name:base_report_to_printer.printing_menu
+#: model_terms:ir.ui.view,arch_db:base_report_to_printer.view_users_form
+msgid "Printing"
+msgstr "Impression en cours"
```

Comment:
Impressions

Reply:
As far as I know translations are not handled during migration, should be done in https://translation.odoo-community.org/